docs/docs/components/quickstart/nodejs/actions/README.md

```diff
@@ -0,0 +1,455 @@
+# Quickstart: Action Development
+
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Walkthrough](#walkthrough)
+  - [hello world!](#hello-world)
+  - [hello [name]!](#hello-name)
+  - [Use an npm Package](#use-an-npm-package)
+  - [Use Managed Auth](#use-managed-auth-to-pull-data-from-github-10-mins)
+- [What's Next?](#whats-next)
+
+## Overview
+
+This document is intended for developers who want to author and edit [Pipedream Actions](/components/#actions). After completing this quickstart, you'll understand how to:
+
+- Develop Pipedream components
+- Publish private actions and use them in workflows
+- Use props to capture user input
+- Update an action
+- Use npm packages
+- Use Pipedream managed auth for a 3rd party app
+
+> **NOTE:** If you previously developed actions using Pipedream's UI, we recommend reviewing our [migration guide](/components/migrating/) after completing this quickstart.
```

Comment:
You might check out the custom tip / warning sections, which we use in other places for these kind of notes: https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers .